holoviews/util/__init__.py

```diff
@@ -484,11 +488,18 @@ def builder(cls, spec=None, **kws):
             return Options(spec, **kws)
 
         filtered_keywords = [k for k in completions if k not in cls._no_completion]
-        kws = ', '.join('{opt}=None'.format(opt=opt) for opt in sorted(filtered_keywords))
-        builder.__doc__ = '{element}({kws})'.format(element=element, kws=kws)
+        sorted_kw_set = sorted(set(filtered_keywords))
+        kws = ', '.join('{opt}=None'.format(opt=opt) for opt in sorted_kw_set)
```

Comment:
Doesn't matter much but why not move this into the py2 block?

Reply:
Fixed in dc3c44401